### PR TITLE
venter: Change ordering of comments in serializer

### PR DESCRIPTION
**Pull request by sshivam95**
Changes ordering of comments

### CODE REVIEW

File: venter/migrations/0004_auto_20180923_1236.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('venter', '0003_auto_20180920_1625'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='comment',
+            options={'verbose_name': 'Comment', 'verbose_name_plural': 'Comments'},
```
Comment (pulsejet) [inline venter/migrations/0004_auto_20180923_1236.py:15]: This migration doesn't do anything. Remove it.

File: venter/models.py [hunk 2]
```diff
@@ -61,9 +61,8 @@ class Comment(models.Model):
     complaint = models.ForeignKey(Complaints, on_delete=models.CASCADE, related_name='comments')
 
     class Meta:
-        verbose_name = 'Comment'
-        verbose_name_plural = 'Comments'
-        ordering = ('-time',)
+        verbose_name = "Comment"
+        verbose_name_plural = "Comments"
```
Comment (pulsejet) [inline venter/models.py]: Use single inverted commas everywhere. This change should be just one line.

**Review by pulsejet — CHANGES_REQUESTED** [review]
Why do I see so many commits here? I think you need to make a new branch rebased on master.

File: venter/migrations/0004_auto_20180923_1236.py (hunk 1 above)
Comment (sshivam95) [reply venter/migrations/0004_auto_20180923_1236.py:15]: In the initial migration, there it is mentioned in the options of the comment model that it is being ordered in '-time', I have just removed it

**Review by pulsejet — APPROVED** [review]

**Pull request merged**